src/dropdown/list/item/DropdownListItem.tsx

```diff
@@ -0,0 +1,115 @@
+import "./_dropdown-list-item.scss";
+
+import React, {Fragment} from "react";
+import classNames from "classnames";
+
+export interface DropdownOption<Id = string, Context = any> {
+  id: Id;
+  customClassName?: string;
+  title: string;
+  CustomContent?: JSX.Element;
+  icon?: React.ReactNode;
+  subtitle?: string;
+  context?: Context;
```

Comment:
I think when typing the props, it is better to list required ones before the optional ones.
```suggestion
  id: Id;
  title: string;
  customClassName?: string;
  CustomContent?: JSX.Element;
  icon?: React.ReactNode;
  subtitle?: string;
  context?: Context;
```